app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -397,6 +397,8 @@ class LuciBuildService {
         userData: json.encode(userData),
       ),
     ));
+    final String buildUrl = 'https://ci.chromium.org/ui/b/${scheduleBuild.id}';
+    await githubChecksUtil.updateCheckRun(config, slug, githubCheckRun, detailsUrl: buildUrl);
     return true;
```

Comment:
Not necessary for this PR, but we should start returning the builds instead of true so we can pass this information around

Reply:
Added https://github.com/flutter/flutter/issues/83945 to not forget updating the methods to return a Build object.